appveyor.yml

```diff
@@ -0,0 +1,24 @@
+﻿version: '{build}'
```

Comment:
Based on a verbal conversation: It is free for opensource and will be used to verify the build on windows machine.